common/src/buffers/in_mem/recv.rs

```diff
@@ -32,6 +33,17 @@ impl Default for ChunkBuffer {
     }
 }
 
+#[derive(Clone, Default)]
+pub struct InMemoryReceivingBufferConfig;
+
+#[async_trait]
+impl ReceivingBufferConfig for InMemoryReceivingBufferConfig {
+    type Buffer = InMemoryReceivingBuffer;
+    async fn create(&self) -> Result<InMemoryReceivingBuffer, DenimBufferError> {
+        Ok(InMemoryReceivingBuffer::default())
+    }
+}
```

Comment:
Should be used in e2e tests

Reply:
this will be done when i make e2e in my next pull request